VERSION

```diff
@@ -1 +1 @@
-2.2.7
\ No newline at end of file
+2.2.7.pre8.1
```

Comment:
I'm not sure if there is any processing done on this value that expects a `x.y.z` format, to be on the safe side it might be better to change this to something like `2.2.7-pre8-1`

Reply:
Good point. I will change it and see if anything fails :)